.github/workflows/maven.yml

```diff
@@ -3,13 +3,16 @@
 
 name: Waltz Build (pg)
 
-on: push
-      
+on:
+  push:
+    tags: ['*']
+    branches: ['*']
+
```

Comment:
their original was just `on: push` but since I deleted new workflow (Docker) and merged our steps here at their original I needed to change this too in order to execute a release
which now I ask myself...is it possible that did `on: push` and release was triggered too, even with condition to run on tags?

Reply:
I believe that `on: push` will be triggered for any push (branch or tag)

Reply:
tested, works as expected....thank you for this!
https://github.com/ljubon/waltz/runs/4460171796?check_suite_focus=true
https://github.com/ljubon/waltz/runs/4460168450?check_suite_focus=true